src/pages/resultsView/plots/PlotsTabUtils.tsx

```diff
@@ -289,7 +289,8 @@ export function getColoringMenuOptionValue(
     option: Omit<ColoringMenuOmnibarOption, 'value'>
 ) {
     return `${option.info.entrezGeneId}_${JSON.stringify(
-        option.info.clinicalAttribute
+        option.info.clinicalAttribute,
+        ['clinicalAttributeId', 'patientAttribute', 'studyId']
```

Comment:
@alisman @tmazor @jjgao I should have done this from the start. But this will cause plots tab URLs which people have saved to no longer point to the "color by" setting that they used to point to. Is that acceptable? However this will also make the URL much more concise.